api/core/api.proto

```diff
@@ -114,13 +118,15 @@ message ResultData {
 // {
 //   "serviceID": "__SERVICE_ID__",
 //   "taskKey":   "__TASK_KEY__",
-//   "inputData": "{\"foo\":\"bar\"}"
+//   "inputData": "{\"foo\":\"bar\"}",
+//   "tags":      ["executionX", "test"]
 // }
 // ```
 message ExecuteTaskRequest {
-  string serviceID = 1; // The Service ID. Generated when using the [`DeployService` API](#deployservice).
-  string taskKey = 2;   // The task's key to execute.
-  string inputData = 3; // The inputs of the task to execute, encoded in JSON.
+  string serviceID = 1;     // The Service ID. Generated when using the [`DeployService` API](#deployservice).
+  string taskKey = 2;       // The task's key to execute.
+  string inputData = 3;     // The inputs of the task to execute, encoded in JSON.
+  repeated string tags = 4; // The list of tags to associate to the execution
```

Comment:
My argument to name it `executionTags` is to be as explicit as possible without needing anymore context.
Also, we name everything by using 2 words. So let's continue this way.

That's what we did for all other parameters. Eg:

https://github.com/mesg-foundation/core/blob/5f98a3f23a1653664fbf8e3a61bbbb1027cd378d/api/core/api.proto#L70-L73

https://github.com/mesg-foundation/core/blob/5f98a3f23a1653664fbf8e3a61bbbb1027cd378d/api/service/api.proto#L39-L43

Reply:
for the rest of the conversation
https://github.com/mesg-foundation/core/pull/362/files/12d3d3cba19155ab54cb58118d3b4809656957e6#r209819975

I would love to have other feedbacks on that @ilgooz @krhubert ?

Naming problem again :)

For the naming of the tag parameter sent when executing a task or listening for the result of a task.

- tags (simple, related to the context, can be reused in any other kind of resources to add metadata)
- executionTags (strongly coupled to the execution, gives a lot of details without knowing the context, we often have 2 words attribute)

What do you think ?